backend/utils/authutils/tvault.go

```diff
@@ -0,0 +1,89 @@
+package authutils
+
+import (
+	"Tella-Desktop/backend/utils/constants"
+	"io"
+	"os"
+)
+
+// TVault helper methods -> move this to utils
+func WriteTVaultHeader(salt, encryptDBKey []byte) error {
```

Comment:
another note: this version of `WriteTVaultHeader` seems to only service the use case of the initial creation of the tvault header. 

in the future, to e.g. support the database recovery flow, you will need a method to overwrite parts of the header, or for changing the header's database version. for example i think file.Write() just appends to the file handler. so: if you reuse this method for those future use cases you will have an issue. consider renaming this method to like, `CreateTVaultHeader` or `InitializeTVaultHeader` to prevent any future mishaps?